tests/openssl_config_DIRACDockerDevBox_usr

```diff
@@ -0,0 +1,23 @@
+[ req ]
+default_bits           = 1024
+encrypt_key            = yes
+distinguished_name     = req_dn
+prompt                 = no
+x509_extensions        = usr_cert
+
+[ req_dn ]
+C                      = ch
+O                      = DIRAC
+OU                     = DIRAC CI
+CN                     = DIRACDockerDevBox
+emailAddress           = DIRACDockerDevBox@diracgrid.org
+
+[ usr_cert ]
+# Extensions for client certificates (`man x509v3_config`).
+basicConstraints = CA:FALSE
+nsCertType = client, email
```

Comment:
I am not entirely sure that it will work for a host. I *think* it also needs the server Usage. It will work in DIRAC, but that's because we do not have proper TLS protocol implemented. It will fail using any reasonable implementation (including Tornado)